### DISCUSSION TITLE
Should we do a new release soon?

### DISCUSSION THREAD

**Discussion by damithc** [MarkBind/markbind#1526] [General]
Not strictly required but can consider doing a release if and when the current master branch reaches a state that is suitable for a release. @ang-zeyu what do you think?

**Comment by ang-zeyu** [comment]
Since its a major release, I'm thinking roughly a month more to let 1 / 2 more major features come in hopefully (ideally versioning) =P

https://github.com/MarkBind/markbind/milestone/72?closed=1

Should give more time to deal with all deprecation warnings as well

**Comment by damithc** [comment]
@wxwxwxwx9 @ryoarmanda @jonahtanjz @raysonkoh Pls pitch in if possible to sort out the V3 issues that surfaced recently.

**Comment by damithc** [comment]
@wxwxwxwx9 @ryoarmanda @jonahtanjz @raysonkoh  Guys, let's aim to fix up any V3 high-priority issues and do a minor release by this weekend.
